plugins/node/opentelemetry-instrumentation-mysql/src/mysql.ts

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { diag, Span, SpanKind, SpanStatusCode } from '@opentelemetry/api';
+import {
+  InstrumentationBase,
+  InstrumentationNodeModuleDefinition,
+  isWrapped,
+} from '@opentelemetry/instrumentation';
+import { DatabaseAttribute } from '@opentelemetry/semantic-conventions';
+import type * as mysqlTypes from 'mysql';
+import { MysqlInstrumentationConfig } from './types';
+import { getConnectionAttributes, getDbStatement, getSpanName } from './utils';
+import { VERSION } from './version';
+
+type formatType = typeof mysqlTypes.format;
+
+export class MysqlInstrumentation extends InstrumentationBase<
```

Comment:
Previously we had `MySQLPlugin`, the db itself has a name `MySQL` not `Mysql`, it is the same with mongoDB, I think we should stick to name `MySQLInstrumentation`

Reply:
Done